content/public/common/content_switches.h

```diff
@@ -224,6 +224,7 @@ CONTENT_EXPORT extern const char kUseFakeUIForFedCM[];
 CONTENT_EXPORT extern const char kUseFakeUIForMediaStream[];
 CONTENT_EXPORT extern const char kVideoImageTextureTarget[];
 CONTENT_EXPORT extern const char kUseMobileUserAgent[];
+CONTENT_EXPORT extern const char kUseCobaltUserAgent[];
```

Comment:
Same comment as elsewhere - this should live under /cobalt, be runtime-gated or as a last resort, #ifdef gated for Cobalt.

Reply:
I can remove this switch with the change I proposed above. Working on it now.